README.md

```diff
@@ -231,8 +232,57 @@ final config = {
 };
 ```
 
+## Format on save
+dart_dev can be used to facilate formatting on save inside of JetBrains IDEs. For setup instructions, see below.
+
+__NOTE:__ This functionality is reliant upon the `pub run dart_dev` process running at least version `3.6.0` of this tool. Ensure that your `pubspec.yaml` specifies an appropriate version range, otherwise the method described below will not work.
```

Comment:
Similar to the ORF change a while back, this min version assumes we'll bump ddev a minor for adding a new API